### DISCUSSION TITLE
How to use BatchService (maybe hacked account)?

### DISCUSSION THREAD

**Discussion by saikoml** [pocketbase/pocketbase#5587] [Q&A]
How to use the createBatch from js? The example from https://www.npmjs.com/package/pocketbase/v/0.22.0-rc#user-content-batchservice return 404. 

I also compared the npm readme vs the github repo https://github.com/pocketbase/js-sdk and there is no BatchService example at all? Is your account hacked or something?

**Comment by ganigeorgiev** [comment]
The JS SDK BatchService works only with PoketBase v0.23.0 which is not officially released yet.

The npm JS SDK v0.22.0-rc version readme differs from the repo master's readme because the JS SDK release is tagged from its [`develop` branch](https://github.com/pocketbase/js-sdk/tree/develop).

There is no official GitHub release yet because I'm still not ready with the upgrade guide.

The v0.23.0 changelog ended up ~3k lines (combined changes from the WeB APIs, Go and JSVM) and I realized that no one is going to read a giant markdown text file so I'm currently splitting and restructuring it in several html dedicated upgrade pages that will be available in the web site and will look something like:
![upgrade_screenshot](https://i.imgur.com/Ah544tn.png)

There will be public GitHub release candidate announcement once the upgrade guide is ready but this might take some time so please be patient.

**Reply by saikoml** [reply]
But why not publish it as it is? Isn't the "rc" exactly for incomplete things?

**Reply by ganigeorgiev** [reply]
Because it is not ready yet. There is no much point of release candidate if no one will be able to test and experiment with it.

If you are really that impatient you can build the PocketBase [`develop` branch](https://github.com/pocketbase/pocketbase/tree/develop) on your own but I recommend to wait for the official release candidate, especially if you plan to upgrade an existing PocketBase application.

**Reply by saikoml** [reply]
Then why tagging with "-rc" the sdk?

**Reply by ganigeorgiev** [reply]
Because I wanted to run some experiments with Presentator and to be able to pass the repo's new GitHub action tests.

In any case this type of discussion is pointless. Either wait for the release or use something else and move on.
